draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -454,7 +436,42 @@ about individual client data.
 
 # Applying Privacy Partioning
 
-TODO: talk about how to think about new protocols that are using partitioning, the models that we derive from looking at the existing ones, how you recognize a protocol that's using this, and what tools do we have to evaluate it
+Applying privacy partitioning to an existing or new system or protocol requires the following steps:
+
+1. Identify the types of information used or exposed in a system or protocol, some of which can be used to identify a user or correlate to other contexts.
+1. Partition different types of user-identifying or correlatable information across contexts to minimize the amount of
+user-identifying or correlatable information in any single context.
```

Comment:
> minimize the amount of  user-identifying or correlatable information in any single context.
Per discussion slack, I actually think this is not fully correct and also not what we say earlier. Earlier we talk about separate the who from the what and minimising amount of information needed to fulfill a function to only reveal user-specific information to parties that need access to the information for their intended purpose.

Reply:
(Is this now resolved?)